decidim-admin/app/forms/decidim/admin/impersonate_managed_user_form.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    # A form object used to impersonate managed users from the admin dashboard.
+    #
+    # This form will contain a dynamic attribute for the user authorization.
+    # This authorization will be selected by the admin user if more than one exists.
+    class ImpersonateManagedUserForm < Form
+      def initialize(attributes)
+        extend(Virtus.model)
```

Comment:
@josepjaume I saw that as an example to how to add dynamic attributes to a form: https://github.com/solnic/virtus/issues/122#issuecomment-9925765

I think it might work if I extend the class directly. Should I try it?